.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-max-line-length = 88
+max-line-length = 150
```

Comment:
I think this is too much, @TheAyeshaSiddiqa. Can you make it 120 maximum and update the texts?